RecoEgamma/PhotonIdentification/plugins/PhotonIDValueMapProducer.cc

```diff
@@ -42,13 +42,13 @@ class PhotonIDValueMapProducer : public edm::stream::EDProducer<> {
   public:
   
   explicit PhotonIDValueMapProducer(const edm::ParameterSet&);
-  ~PhotonIDValueMapProducer();
```

Comment:
changes in this file are in the release already (the same commit IDs and in the same order).

Reply:
so?